conversant/prompt_chatbot.py

```diff
@@ -118,6 +128,52 @@ def latest_prompt(self) -> str:
         """
         return self.prompt_history[-1]
 
+    def _update_max_context_examples(
+        self, prompt_size: int, max_context_examples: int
+    ) -> int:
+        """Adjust max_context_examples until a possible prompt size.
+
+        if this is not possible, send an error message.
+
+        Args:
+            prompt_size (int): Number of tokens of the prompt
+            max_context_examples (int): The length of the chat history for
+            the chatbot to use in reply.
+
+        Returns:
+           int: updated max_context_examples
+        """
+        # Store original values
+        original_size = prompt_size
+        # If the size of chat_history is smaller than max_context_examples
+        # the value of the variable is already updated with the size value
+        trimmed_max_examples = min(len(self.chat_history), max_context_examples)
+
+        # Reduce max_context_examples until the number of token of the prompt
+        # is less than maximum or reaches 1
+        for size in self.prompt_size_history[-max_context_examples:]:
```

Comment:
I think this might lead to a bug similar to https://github.com/cohere-ai/sandbox-conversant-lib/pull/12

If max_context_examples is 0, then `self.prompt_size_history[-0:]` returns the entire `self.prompt_size_history`

Reply:
Good catch!! I'll add a check if max_context_examples is bigger than to 0 run the loop, because if is not O should just raise the error because it's not possible to reduce